### PR TITLE
Simply saying hi to the coolest guy ever tee hee, Darion

### PR DESCRIPTION
**Pull request by jakyle**
hey, can you merge this pls? ty :)

### CODE REVIEW

**Comment by DarionThompson** [comment]
What is this feature ? Did you get this approved ?

**Comment by jakyle** [comment]
yeah :)

File: Views/Home/HiDarion.cshtml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+}
```
Comment (DarionThompson) [inline Views/Home/HiDarion.cshtml]: Extra white space 
Ill let it slide this time

File: Views/Home/HiDarion.cshtml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+
+<p>Hi Darion 😙</p>
```
Comment (jakyle) [inline Views/Home/HiDarion.cshtml]: did our designer approve this emoji?

File: Controllers/HomeController.cs [hunk 3]
```diff
@@ -28,6 +28,11 @@ public IActionResult Privacy()
             return View();
         }
 
+        public IActionResult HiDarion()
+        {
```
Comment (DarionThompson) [inline Controllers/HomeController.cs:32]: Empty View ?

**Review by DarionThompson — APPROVED** [review]
Great Job !!

**Pull request merged**